meinberlin/settings/base.py

```diff
@@ -228,6 +228,8 @@
 ACCOUNT_LOGIN_ON_EMAIL_CONFIRMATION = True
 ACCOUNT_LOGIN_ON_PASSWORD_RESET = True
 
+LOGIN_URL = 'account_login'
+LOGOUT_URL = 'account_logout'
```

Comment:
There is no LOGOUT_URL setting and it's nowhere used. https://tree.taiga.io/project/ggr-meinberlinteam/us/1009?no-milestone=1